community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -783,6 +783,13 @@ object projects:
     dependencies = () => List(cats, disciplineMunit)
   )
 
+  lazy val http4s = SbtCommunityProject(
+    project = "http4s",
+    sbtTestCommand = "set ThisBuild/tlFatalWarnings := false; rootJVM/test",
+    sbtPublishCommand = "set ThisBuild/tlFatalWarnings := false; publishLocal",
+    dependencies = () => Nil
```

Comment:
I omitted the dependencies for now b/c http4s depends on newer cats/effect/fs2 etc. versions than what's currently offered in the build.

Reply:
I believe this approach is quite fragile and has only succeeded here because `https` has been added to `CommunityBuildTestC` whereas the rest of the Typelevel stack is in `CommunityBuildTestB`.

If another project that depends on cats/effect/fs2 is ever added to `CommunityBuildC`, or `https` is moved to `CommunityBuildB`, then this strategy falls apart, as the community build injects dependency overrides for all previously built projects in each partition.

The right thing to do here IMO is to bump the dependencies in the community build.  I'll open a PR to do so, unless there's some reason we can't (have the issues with `sbt-spiewak` been resolved)?

Reply:
Thanks for your comment. Ah, I wasn't aware that was the difference between the community builds. I agree, this is probably not the right approach esp. for long-term.

Unfortunately the issues with sbt-spiewak have not been resolved. http4s is now using sbt-typelevel, as is fs2, so it should be possible to update those. cats-effect is still using sbt-spiewak unfortunately. Cats may have problems because of https://github.com/typelevel/cats/blob/9ab4a6e00737d074339e0b2c124c832aebc47c88/build.sbt#L6 but that can be fixed.

Reply:
cc @djspiewak

Reply:
Good news, it seems that Cats Effect is not currently referencing the `isDotty` setting in its `build.sbt`. So I think it should be fine.

I opened https://github.com/typelevel/cats/pull/4119 for Cats.

Reply:
@griggt actually it looks like the Cats already in the community build has the `isDotty` setting defined in its build without conflicts. So it must be okay. I think you should be able to update all the dependencies 👍 

https://github.com/dotty-staging/cats/blob/878472d7bff4c3bfec8a265782c1e0d6a3147541/build.sbt#L6